.pipelines/prod-release.yml

```diff
@@ -6,6 +6,13 @@ parameters:
 - name: vsoConfigBuildID
 - name: vsoDeployerBuildID
 
+resources:
```

Comment:
Uses `AROGIT` service connection for readonly access to msazure organization to pull the config from `AzureRedHatOpenShift/RP-Config` repository.